README.md

```diff
@@ -255,6 +255,7 @@ Split your app module into different small modules, and give those modules as de
 
 <br/>Read more [here](https://medium.com/swlh/modularization-by-feature-and-layer-with-android-architecture-components-80bf317d737).
 
+28) Avoid using too many Base classes. Using base classes everywhere makes a tight web in your codebase, which later makes it hard to do changes in the code. If still necessary, create and use a standalone function (kotline file).
```

Comment:
Hey Thank you for contributing. Could you please add a few example or details for anyone to undestand it better.

Reply:
Hi, I have added a detailed explaination along with some references.